app/Http/Controllers/PublicJoinController.php

```diff
@@ -56,7 +56,7 @@ public function store(Request $request)
 
         $this->validate($request, [
             'name' => 'required|max:255',
-            'age' => 'required|integer|min:' . config('app.min_age'),
+            'age' => 'required|integer',
```

Comment:
Might want to change it to `required|integer|gt:0`, but doesn't really matter much.